src/features/network/components/network-configs-table.tsx

```diff
@@ -75,10 +78,20 @@ const tableColumns: ColumnDef<NetworkConfigWithId>[] = [
     meta: { className: 'hidden md:table-cell' },
     accessorFn: (item) => `${trimCharacterFromEnd(item.indexer.server, '/')}:${item.indexer.port}`,
   },
+  {
+    id: 'activate',
+    header: '',
+    meta: { className: `w-[${ICON_BUTTON_SIZE}px]` },
+    accessorFn: (item) => item,
+    cell: (cell) => {
+      const networkConfig = cell.getValue<NetworkConfigWithId>()
+      return <ActivateButton networkConfig={networkConfig} />
+    },
+  },
   {
     id: 'edit',
```

Comment:
I think it may be better to bring these buttons into a single "Actions" (no need for a visible header) column and reduce the padding a little between the buttons. The icon also feels a bit big for the button. The app interface page has an edit and delete icon button, when you have. The styling and padding should match this.

<img width="816" alt="Screenshot 2024-11-22 at 10 41 55 PM" src="https://github.com/user-attachments/assets/f62c2fa2-d89a-41db-8b55-83a82bbb532b">

Also when reducing the screen width, the buttons appear as below.
<img width="607" alt="Screenshot 2024-11-22 at 10 42 37 PM" src="https://github.com/user-attachments/assets/ba735fb8-7798-41b8-b887-4524247397d0">

Reply:
Good call. Made the change. I also noticed that the cells are hidden on small screen sizes but the table headers are not as shown in the screenshot above. I applied the same `meta.className` to the headers in the DataTable as well so they get hidden on small screen sizes. Are there cases where you want the `meta.className` only to applied to the cells and not the header?

<img width="490" alt="image" src="https://github.com/user-attachments/assets/f223c52f-ea9f-4d75-a437-a3b677fdfae1">

Reply:
Yeah I think there may be. I can check and confirm.

Reply:
Also I don't think there is any need to have the "Actions" text visible.

Reply:
I removed the actions header title. I'll wait on confirmation re: meta.className. If there is a need to differentiate the two, perhaps we can use `meta.className` and `meta.headerClassName` although that might be redundant if we need to make them the same most of the time.

Reply:
I've checked and this change is all good.